src/main/java/spotify/api/enums/QueryType.java

```diff
@@ -1,5 +1,15 @@
 package spotify.api.enums;
 
+/**
+ * This class contains suitable query types used for the search query API endpoint.
+ * <p>
+ * For further information see: <a href="https://developer.spotify.com/documentation/web-api/reference/search/search/">
+ * Search</a>.
+ *
+ * @author Jiankai Zheng (jk.zheng@hotmail.com)
+ * @since 1.0.0
```

Comment:
Not added in 1.0.0

Reply:
You say not added in 1.0.0, but I show it was added in 1.0.0? Unless I am reading the releases wrong? I went to [](https://github.com/jzheng2017/spotify-web-api-wrapper/compare/1.0.0...main) and saw it there

Reply:
What you're comparing right now is all the changes between version 1.0.0 and the current main branch. This class was added somwhere between those two version but not in version 1.0.0